### PR TITLE
Scorewatch June Week 4

### PR DESCRIPTION
**Pull request by ThePooN**

### CODE REVIEW

File: news/2017-07-03-scorewatch-june-week-4.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+layout: post
+title: "Scorewatch: June Week 4"
+permalink: http://osunews.tumblr.com/post/162432090863/scorewatch-june-week-4
+date: 2017-07-03 02:00:00 +0200
+tumblr_url: http://osunews.tumblr.com/post/162432090863/scorewatch-june-week-4
+---
+ 
+Many impressive first-ever-done scores, first places and more in this week’s article! 
+ 
+![](https://puu.sh/wA4Nj/d299c2b68d.png)
```
Comment (Ephemeralis) [inline news/2017-07-03-scorewatch-june-week-4.md]: Add to the repo under ``/wiki/shared/news`` following appropriate folder nomenclature for the filename

File: news/2017-07-03-scorewatch-june-week-4.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title: "Scorewatch: June Week 4"
+permalink: http://osunews.tumblr.com/post/162432090863/scorewatch-june-week-4
+date: 2017-07-03 02:00:00 +0200
+tumblr_url: http://osunews.tumblr.com/post/162432090863/scorewatch-june-week-4
+---
+ 
+Many impressive first-ever-done scores, first places and more in this week’s article!
```
Comment (Ephemeralis) [inline news/2017-07-03-scorewatch-june-week-4.md]: This is a bit short. Add something in briefly covering the major scores on the maps by name or something? Two lines is fine, but one is way too short.

File: news/2017-07-03-scorewatch-june-week-4.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/dOTlAs9lK4I" frameborder="0" allowfullscreen></iframe></br>
+
+[-GN](https://osu.ppy.sh/u/895581) might have churned out the craziest osu! EZHD play of all time, pulling off an absolutely terrifying 96.56% EZHD FC on [Nico Nico Douga - BARUSA of MIKOSU](https://osu.ppy.sh/b/24722) (which is 9.53*!), taking the No. 1 spot on the map live at Dreamhack. This would be worth approximately 707pp. What else can be better than for you to watch this amazing play yourself, together with the live chat, straight from -GN’s YouTube channel!
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/MyTOUzA-sS8" frameborder="0" allowfullscreen></iframe></br>
+
+[Dusk](https://osu.ppy.sh/u/533210) is serious about cementing No. 1 Global as he continues his osu!catch score runs with an absurd HR SS on [LeaF - Evanescent](https://osu.ppy.sh/b/1009780) to gain a whopping 743pp and the No. 2 spot on the map, second only to [[224]Hyperw7](https://osu.ppy.sh/u/4158549) who was 1 100 short of SSing it himself. Sit back, relax and have a look at Rowlet sprinting across the screen with joy.
```
Comment (Ephemeralis) [inline news/2017-07-03-scorewatch-june-week-4.md]: ``No . 1 Global`` -> his number one place in the global rankings
``No. 2`` -> number two
comma after hyperw7 hyperlink

maybe specify "rowlet catcher" or something so people don't get confused as to wtf a rowlet is

File: news/2017-07-03-scorewatch-june-week-4.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/zqidKHSJ2-Q" frameborder="0" allowfullscreen></iframe></br>
+
+We haven’t shown [jakads](https://osu.ppy.sh/u/259972) in the Scorewatch for a while now, so here he is, taking [sakuzyo - Altale](https://osu.ppy.sh/b/767311&m=3) and slapping on DT to pull off a crazy 99.87% FC on the hardest difficulty worth 677pp. Just enjoy some intense keyboard action along with jakads’ interesting sounds.
```
Comment (Ephemeralis) [inline news/2017-07-03-scorewatch-june-week-4.md]: ``sounds`` -> reactions

File: news/2017-07-03-scorewatch-june-week-4.md (hunk 2 above)
Comment (Xyloo) [inline news/2017-07-03-scorewatch-june-week-4.md]: Many impressive first-ever-done scores, a lot of EZ and DT mod, a TAG4 FC, several first places, a taiko scoreshow and more in this week's article!

is this any better?

Comment (Ephemeralis) [inline news/2017-07-03-scorewatch-june-week-4.md]: Getting there, just throw some map names into it as well as those tend to attract people the most.

Comment (Xyloo) [inline news/2017-07-03-scorewatch-june-week-4.md]: Many impressive first-ever-done scores on maps like BARUSA of MIKOSU, a lot of EZ and DT mod action, several first places, a taiko scoreshow and more in this week's article!

i'm not that good at writing captions like these

Comment (Ephemeralis) [inline news/2017-07-03-scorewatch-june-week-4.md]: That seems fine to me, much better. Sub ``article`` for something like ``in the final issue of Scorewatch for June!`` maybe

Comment (Xyloo) [inline news/2017-07-03-scorewatch-june-week-4.md]: Many impressive first-ever-done scores on maps like BARUSA of MIKOSU, a lot of EZ and DT mod action, several first places, a taiko scoreshow and more in the final issue of Scorewatch for June!

@ThePooN i guess we can push with that

Comment (ThePooN) [inline news/2017-07-03-scorewatch-june-week-4.md]: Hi I'm ZePooNzz and I haz just pushedzz all the fixezzzz but GitHubz wont mark this az rezzzzolved

**Pull request merged**